src/test/resources/application.yml

```
@@ -0,0 +1,3 @@
+spring:
+  config:
+    import: optional:file:.env[.properties]
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)